src/Common/utils.ts

```diff
@@ -24,7 +24,7 @@ export interface ClassicAddress {
   test: boolean
 }
 
-abstract class Utils {
+const utils = {
```

Comment:
Change from a class to an object that holds functions.

This is the proper fix for the [no-extraneous-class](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-extraneous-class.md) rule. I did ask about [using abstract classes for this problem](https://github.com/typescript-eslint/typescript-eslint/issues/2015), but the consensus seems to be to just use EcmaScript modules.

It's a little odd that we're exporting an object as opposed to the individual functions, but this keeps us consistent with the old way of exporting the class.